### PR TITLE
Move funcs of create.go to e2e/storage

### PR DESCRIPTION
**Pull request by clarklee92**
**What type of PR is this?**
 /kind cleanup

**What this PR does / why we need it**:
Move functions in create.go to module e2e/storage

**Which issue(s) this PR fixes**:

Ref: #84380

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

/assign @oomichi

### CODE REVIEW

File: test/e2e/storage/BUILD [hunk 1]
```diff
@@ -3,6 +3,7 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
 go_library(
     name = "go_default_library",
     srcs = [
+        "create.go",
```
Comment (oomichi) [inline test/e2e/storage/BUILD]: This go file itself is not e2e test, just a utility.
In addition, `k8s.io/kubernetes/test/e2e/storage/utils` exists already.
So how about moving this code into `k8s.io/kubernetes/test/e2e/storage/utils` ?

File: test/e2e/storage/drivers/csi.go [hunk 2]
```diff
@@ -41,6 +41,7 @@ import (
 	"time"
 
 	"github.com/onsi/ginkgo"
+
```
Comment (oomichi) [inline test/e2e/storage/drivers/csi.go]: unrelated change.

File: test/e2e/storage/external/external.go [hunk 3]
```diff
@@ -20,6 +20,8 @@ import (
 	"flag"
 	"io/ioutil"
 
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
```
Comment (oomichi) [inline test/e2e/storage/external/external.go]: ditto: unrelated change

**Review by alejandrox1 — COMMENTED** [review]
wow, this is really cool @clarklee92 !
/ok-to-test
/lgtm
/priority important-soon
/milestone v1.17

**Comment by alejandrox1** [comment]
/lgtm cancel
/hold
just to quiet the ci bot

File: test/e2e/storage/BUILD (hunk 1 above)
Comment (clarklee92) [reply test/e2e/storage/BUILD]: Thanks for you suggestion, sounds more reasonable.

File: test/e2e/storage/drivers/csi.go (hunk 2 above)
Comment (clarklee92) [reply test/e2e/storage/drivers/csi.go]: Ok, I will roll back that. I thought the imports not formated and modified together.

File: test/e2e/storage/external/external.go (hunk 3 above)
Comment (clarklee92) [reply test/e2e/storage/external/external.go]: Already force pushed new version.

**Comment by oomichi** [comment]
Thanks for doing this, very clean way for me.

/lgtm

**Comment by clarklee92** [comment]
> Thanks for doing this, very clean way for me.
> 
> /lgtm

Please tag again, a conflict solved after some new merge. 😁

**Comment by oomichi** [comment]
@clarklee92 Thanks for updating, but unfortunately this PR seems to be created with something wrong as k8s-ci-robot said as https://github.com/kubernetes/kubernetes/pull/84463#issuecomment-547713396
So please fix this issue by creating the PR from clean k/k repository.

**Comment by clarklee92** [comment]
> @clarklee92 Thanks for updating, but unfortunately this PR seems to be created with something wrong as k8s-ci-robot said as [#84463 (comment)](https://github.com/kubernetes/kubernetes/pull/84463#issuecomment-547713396)
> So please fix this issue by creating the PR from clean k/k repository.

Fixed, please.

**Review by oomichi — COMMENTED** [review]
Nice work, thanks!

/lgtm
/approve

**Comment by oomichi** [comment]
@clarklee92 Could you rebase this PR for the latest master?

/lgtm cancel
/hold cancel
